packages/function-tree/src/index.js

```diff
@@ -191,6 +191,11 @@ export class FunctionTree extends EventEmitter {
     this.runTree.on = this.on.bind(this)
     this.runTree.once = this.once.bind(this)
     this.runTree.off = this.removeListener.bind(this)
+
+    // When not extended
+    if (!returnInstance) {
```

Comment:
behaves as you expect with current published function-tree https://www.webpackbin.com/bins/-KgVhuLwpBVMJB98y5UH